src/main/java/de/tum/in/www1/artemis/service/connectors/BambooService.java

```diff
@@ -850,7 +876,9 @@ private Long calculateScoreForResult(Result result, int skippedTests) {
                         logString.startsWith("[ERROR] For more information about the errors and possible solutions") ||
                         logString.startsWith("[ERROR] Re-run Maven using") ||
                         logString.startsWith("[ERROR] To see the full stack trace of the errors") ||
-                        logString.startsWith("[ERROR] -> [Help 1]")
+                        logString.startsWith("[ERROR] -> [Help 1]") ||
+                        logString.startsWith("Unable to publish artifact") ||
```

Comment:
Is that something that is caused by spotbugs or by Bamboo?
So if a different tool gets added, do we also have to change something here?

Reply:
This logged by bamboo itself if an artifact can't be created. Happens for example if a compilation error occurs and and no Bytecode is available for analysis.

Reply:
Great!